fdbserver/include/fdbserver/IKeyValueStore.h

```diff
@@ -82,6 +82,9 @@ class IKeyValueStore : public IClosable {
 	// Removes a key range from KVS and returns a list of empty physical shards after the removal.
 	virtual std::vector<std::string> removeRange(KeyRangeRef range) { return std::vector<std::string>(); }
 
+	// Replace the specified range, the default implementation ignores `blockRange` and writes the key one by one.
+	virtual Future<Void> replaceRange(KeyRange blockRange, Standalone<VectorRef<KeyValueRef>> blockData);
```

Comment:
The default implementation should clear the range, otherwise it is not fulfilling the named contract.

Also, let's just use `range` and `data` or `kvPairs` or `records` etc for the argument names.  These aren't really "blocks", that would be a description more for `replaceRangeNative()` if we build that.